openshift/templates/pims-db/mssql-deploy.yaml

```diff
@@ -149,21 +169,51 @@ parameters:
     description: A unique identifier to allow for multiple instances (i.e. "-01").
     value: ""
 
-  - name: SQL_SERVER_RUNTIME_IMAGE
-    displayName: Microsoft SQL Server image
-    description: The Microsoft SQL Server image tag.
+  - name: IMAGE_TAG
+    displayName: Output Image Tag
+    description: The tag given to the built image.
     required: true
-    value: mssql-rhel-server:2019-latest
+    value: latest
 
+  - name: MSSQL_PID
+    displayName: The Microsoft SQL Server Edition
+    description: The edition of the database to run [Developer|Express|Standard|Enterprise|EnterpriseCore]
+    required: true
+    value: Developer
   - name: MSSQL_SA_PASSWORD
     displayName: Microsoft SQL Server Password
     description: Microsoft SQL Server sa password
     from: '[\w]{5}_[\w]{15}!'
     generate: expression
+  - name: TIMEZONE
```

Comment:
Since we are using DateTimes and not DateTimeOffsets I'm not sure we can reasonably support this:
https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime

I think we are restricted to running the DB in local or UTC only, unless none of our datetimes are dependent on timezone (ie. all datetimes are used to track dates or times only).

Reply:
This is unrelated.  The image has an option to set the timezone, I've imply added it as configuration option.  

We can choose whatever date datatype we want when storing dates.